### PR TITLE
[WebDriver BiDi] Fix test for window.prompt opened

### PR DESCRIPTION
**Pull request by Lightning00Blade**
In the HTML IDL - https://html.spec.whatwg.org/#the-window-object the function is defined as:
`prompt(optional DOMString message = "", optional DOMString default = "");`
This means that if no value is passed it get empty string.
I also observed that all values passed to the second parameter get `toString`(ed) (I think this is the algorithm that does it https://webidl.spec.whatwg.org/#js-DOMString),
Verified this behavior for Chrome, Firefox and Safari.

### CODE REVIEW

**Review by OrKoN — APPROVED** [review]

**Review by sadym-chromium — APPROVED** [review]

**Comment by lutien** [comment]
I think we probably then have to update the BiDi spec, because now it assumes that `defaultValue` might be `null`: https://w3c.github.io/webdriver-bidi/#event-browsingContext-userPromptOpened

**Comment by Lightning00Blade** [comment]
@lutien We need the null check in the spec due to other algorithms (`alert`, `confirm`, `beforeunload`), 
not having it send from the HTML spec. And it seems `default value` value for `prompt` is always present and String.
So we want the value to be omitted for the other types but our expectation should that it is always set for `prompt`.

**Review by lutien — APPROVED** [review]
> @lutien We need the null check in the spec due to other algorithms (`alert`, `confirm`, `beforeunload`), not having it send from the HTML spec. And it seems `default value` value for `prompt` is always present and String. So we want the value to be omitted for the other types but our expectation should that it is always set for `prompt`.

Ah, alright, that sounds reasonable to me. Thanks!

**Review by OrKoN — APPROVED** [review]

File: webdriver/tests/bidi/browsing_context/user_prompt_opened/user_prompt_opened.py [hunk 1]
```diff
@@ -49,15 +49,23 @@ async def test_prompt_type(
...
+            "type": prompt_type,
+            "message": text,
+            "defaultValue": ""
+        }
+    else:
+        assert event == {
+            "context": new_tab["context"],
+            "type": prompt_type,
+            "message": text,
+        }
```
Comment (sadym-chromium) [inline webdriver/tests/bidi/browsing_context/user_prompt_opened/user_prompt_opened.py:64]:
```suggestion
    assert event == {
        "context": new_tab["context"],
        "type": prompt_type,
        "message": text,
        **({
            "defaultValue": ""
        } if prompt_type == 'prompt' else {})
    }
```

**Review by sadym-chromium — APPROVED** [review]

**Pull request merged**